### PR TITLE
Trigger rebuild every night

### LINKED ISSUE #4: Trigger daily updates on docker
We need to build this image on Docker daily. Currently the last build was 3 months ago. That means that we are running an outdated version of Psalm. 

One way to solve this issue is to use github action to build periodically and do a CURL request to trigger a dockerhub build.

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix #4 

You need to go to dockerhub. Then "Build", "Configure automatic builds" and add a new "Build trigger". Then you will get a URL that should be used here.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        - cron: '0 3 * * *'
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Trigger build on dockerhub
+        run: |
+          - curl -L -X POST https://hub.docker.com/api/build/v1/source/xxx/trigger/yyy/call/
```
Comment (Nyholm) [inline .github/workflows/build.yml]: @muglug please update this URL.

**Comment by muglug** [comment]
Will do, thanks (probably later on today or tomorrow)

**Comment by Nyholm** [comment]
Thank you

**Comment by muglug** [comment]
Thanks!

**Comment by muglug** [comment]
I guess it needs curl https://github.com/psalm/psalm-github-actions/runs/496852559

**Pull request merged**